test/validators.jl

```diff
@@ -66,6 +70,9 @@
 
         @test_throws DimensionMismatch validate(a[1:10], t)
         @test_throws DimensionMismatch validate(m[1:3, :], t; dims=:cols)
+
+        @test_throws ThresholdError validate(fill(missing, 5), t)
+        @test validate(ones(5), t) == ones(5)
```

Comment:
Do you also want to test the return type?
```suggestion
        @test validate(ones(5), t) == ones(5)
        @test validate(ones(5), t) isa Vector{Float64}
```

Reply:
Yeah, I guess that's probably a good idea.